frontend/src/app/testResults/submissions/Submission.tsx

```diff
@@ -0,0 +1,87 @@
+import moment from "moment";
+import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+
+import { LoadingCard } from "../../commonComponents/LoadingCard/LoadingCard";
+import { useGetUploadSubmissionQuery } from "../../../generated/graphql";
+
+const Submission = () => {
+  const urlParams = useParams();
+  const internalId = urlParams.id || "";
+
+  const organization = useSelector(
+    (state) => (state as any).organization as Organization
+  );
+
+  const { data: submission, loading, error } = useGetUploadSubmissionQuery({
+    fetchPolicy: "no-cache",
+    variables: {
+      id: internalId,
+    },
+  });
+
+  if (loading) {
+    return <LoadingCard />;
+  }
+
+  if (error) {
+    throw error;
+  }
+
+  const transmissionTimestamp = moment(submission?.uploadSubmission.createdAt);
+  const transmissionDate = transmissionTimestamp.format("DD MMM YYYY");
+  const transmissionTime = transmissionTimestamp.format("k:mm");
```

Comment:
This formatting will appear as follows:

> 09 Jun 2022 15:49

This is distinctly **not** the typical datetime formatting we use on SimpleReport, but it is a direct port of the existing ReportStream Uploader UI. I'll leave this as-is for now unless someone feels strongly about not shipping it.